manga_auto_translator/translation.py

```diff
@@ -0,0 +1,38 @@
+from typing import Sequence
+from abc import ABC, abstractmethod
+from enum import Enum
+from manga_auto_translator.data_structure import Scan
+
+
+class TranslationStrategy(ABC):
+    @abstractmethod
+    def run(self, scans: Sequence[Scan], lang_from: str, lang_to: str) -> None:
+        raise NotImplemented()
+
+
+class GoogleTraductionTranslation(TranslationStrategy):
+    def __init__(self) -> None:
+        print('Loading GoogleTraductionTranslation...')
+        
+    def run(self, scans: Sequence[Scan], lang_from: str, lang_to: str) -> None:
+        from deep_translator import GoogleTranslator
+        self.tranlator = GoogleTranslator(source=lang_from, target=lang_to)

```

Comment:
I think the creation of the translation model should be moved to the `__init__` function. Same with the arguments.